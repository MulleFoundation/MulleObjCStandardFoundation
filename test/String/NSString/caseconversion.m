//
//  main.m
//  archiver-test
//
//  Created by Nat! on 19.04.16.
//  Copyright Â© 2016 Mulle kybernetiK. All rights reserved.
//


#import <MulleObjCStandardFoundation/MulleObjCStandardFoundation.h>
//#import "MulleStandaloneObjCFoundation.h"


int main( int argc, const char * argv[])
{
   char       text[] = "VfL Bochum 1848";
   NSString   *s;

   s = [NSString mulleStringWithUTF8Characters:(void *) text
                                        length:sizeof( text)];

   printf( "%s\n", [[s lowercaseString] UTF8String]);
   printf( "%s\n", [[s capitalizedString] UTF8String]);
   printf( "%s\n", [[s uppercaseString] UTF8String]);
   printf( "%s\n", [[s mulleDecapitalizedString] UTF8String]);

   return( 0);
}
