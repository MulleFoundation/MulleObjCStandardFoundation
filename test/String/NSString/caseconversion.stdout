vfl bochum 1848
VfL Bochum 1848
VFL BOCHUM 1848
